com.unity.render-pipelines.universal/ShaderLibrary/Input.hlsl

```diff
@@ -7,6 +7,10 @@
 // Keep in sync with RenderingUtils.useStructuredBuffer
 #define USE_STRUCTURED_BUFFER_FOR_LIGHT_DATA 0
 
+#define RENDERING_LIGHT_LAYERS_MASK (255)
+#define RENDERING_LIGHT_LAYERS_MASK_SHIFT (0)
```

Comment:
why do we have this shift of 0?

Reply:
it is because it's matching HDRP definition for readibility:

in hdrp we have:

#define RENDERING_LIGHT_LAYERS_MASK (255)
#define RENDERING_LIGHT_LAYERS_MASK_SHIFT (0)
#define RENDERING_DECAL_LAYERS_MASK (65280)
#define RENDERING_DECAL_LAYERS_MASK_SHIFT (8)

Reply:
we can have various stuff in rendering layer mask (in the future)

Reply:
I kept this in case more bits of the rendering layer mask are used for other feature (decal?).